project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -249,11 +249,13 @@
             ); ?></div>
         <?php endif; ?>
         <a id="main-content"></a>
-        <?php print render($title_prefix); ?>
-        <?php if (!empty($title)): ?>
-          <h1 class="page-header"><?php print $title; ?></h1>
+        <?php if (!isset($hide_title)): ?>
```

Comment:
`empty` remove the title from `node/add/feed`, and `isset` isn't.
I think I should still use `isset` because `empty` return true if the variable is an empty string, false, array(), NULL, “0?, 0, and an unset variable. and `isset` just when the variable is not null - which is (I think) more appropriate here.